sys/linux/socket_nvme_of_tcp.txt

```diff
@@ -0,0 +1,623 @@
+	apptag		int16
+	appmask		int16
+}
+
+nvme_zone_mgmt_send_cmd {
+	opcode		int8
+	flags		int8
+	command_id	int16
+	nsid		int32
+	cdw2		array[int32, 2]
+	metadata	int64
+	dptr		nvme_data_ptr
+	slba		int64
+	cdw12		int32
+	zsa		int8
+	select_all	int8
+	rsvd13		array[int8, 2]
+	cdw14		array[int32, 2]
+}
+
+nvme_zone_mgmt_recv_cmd {
+	opcode		int8
+	flags		int8
+	command_id	int16
+	nsid		int32
+	rsvd2		array[int64, 2]
+	dptr		nvme_data_ptr
+	slba		int64
+	numd		int32
+	zra		int8
+	zrasf		int8
+	pr		int8
+	rsvd13		int8
+	cdw14		array[int32, 2]
+}
+
+nvme_abort_cmd {
+	opcode		int8
+	flags		int8
+	command_id	int16
+	rsvd1		array[int32, 9]
+	sqid		int16
+	cid		int16
+	rsvd11		array[int32, 5]
+}
+
+nvmf_common_command {
+	opcode		int8
+	resv1		int8
+	command_id	int16
+	fctype		int8
+	resv2		array[int8, 35]
+	ts		array[int8, 24]
+}
+
+nvmf_connect_command {
+	opcode		const[0x7f, int8]
+	resv1		const[0x40, int8]
+	command_id	const[0x0, int16]
+	fctype		const[0x1, int8]
+	resv2		array[int8, 19]
+	dptr		nvme_data_ptr
+	recfmt		int16
+	qid		int16
+	sqsize		int16
+	cattr		int8
+	resv3		int8
+	kato		int32
+	resv4		array[int8, 12]
+}
+
+nvmf_property_set_command {
+	opcode		int8
+	resv1		int8
+	command_id	int16
+	fctype		int8
+	resv2		array[int8, 35]
+	attrib		int8
+	resv3		array[int8, 3]
+	offset		int32
+	value		int64
+	resv4		array[int8, 8]
+}
+
+nvmf_property_get_command {
+	opcode		int8
+	resv1		int8
+	command_id	int16
+	fctype		int8
+	resv2		array[int8, 35]
+	attrib		int8
+	resv3		array[int8, 3]
+	offset		int32
+	resv4		array[int8, 16]
+}
+
+nvmf_auth_common_command {
+	opcode		int8
+	resv1		int8
+	command_id	int16
+	fctype		int8
+	resv2		array[int8, 19]
+	dptr		nvme_data_ptr
+	resv3		int8
+	spsp0		int8
+	spsp1		int8
+	secp		int8
+	al_tl		int32
+	resv4		array[int8, 16]
+}
+
+nvmf_auth_send_command {
+	opcode		int8
+	resv1		int8
+	command_id	int16
+	fctype		int8
+	resv2		array[int8, 19]
+	dptr		nvme_data_ptr
+	resv3		int8
+	spsp0		int8
+	spsp1		int8
+	secp		int8
+	tl		int32
+	resv4		array[int8, 16]
+}
+
+nvmf_auth_receive_command {
+	opcode		int8
+	resv1		int8
+	command_id	int16
+	fctype		int8
+	resv2		array[int8, 19]
+	dptr		nvme_data_ptr
+	resv3		int8
+	spsp0		int8
+	spsp1		int8
+	secp		int8
+	al		int32
+	resv4		array[int8, 16]
+}
+
+nvme_dbbuf {
+	opcode		int8
+	flags		int8
+	command_id	int16
+	rsvd1		array[int32, 5]
+	prp1		int64
+	prp2		int64
+	rsvd12		array[int32, 6]
+}
+
+nvme_directive_cmd {
+	opcode		int8
+	flags		int8
+	command_id	int16
+	nsid		int32
+	rsvd2		array[int64, 2]
+	dptr		nvme_data_ptr
+	numd		int32
+	doper		int8
+	dtype		int8
+	dspec		int16
+	endir		int8
+	tdtype		int8
+	rsvd15		int16
+	rsvd16		array[int32, 3]
+}
+
+#####################
+#####################
+
+nvme_delete_queue {
+	opcode		int8
+	flags		int8
+	command_id	int16
+	rsvd1		array[int32, 9]
+	qid		int16
+	rsvd10		int16
+	rsvd11		array[int32, 5]
+}
+
+nvme_create_sq {
+	opcode		int8
+	flags		int8
+	command_id	int16
+	rsvd1		array[int32, 5]
+	prp1		int64
+	rsvd8		int64
```

Comment:
If the field is reserved and not used by the kernel, you'd better define it as `const[int64, 0]` so that syzkaller won't waste time mutating it.

Here and in all other places.